### ISSUE TITLE
Package is not selected when hovering on "High Contrast on" mode

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt-adafruit#667]
**Repro steps:**
• https://makecode.adafruit.com/beta#editor
• Click "More..." -> "High Contrast on"
• Click "ADVANCED" -> "EXTENSIONS" 
• move cursor mouse on packages

**Expect result:**
On "High Contrast on" mode, when hovering on packages, packages should be selected
![134](https://user-images.githubusercontent.com/23466737/37816915-b8675178-2e6c-11e8-8116-2b27720da0a7.gif)

**Actual result:**
On "High Contrast on" mode, when hovering on packages, package is not selected
![678](https://user-images.githubusercontent.com/23466737/37816965-e4e713be-2e6c-11e8-9250-918827e1def5.gif)
